### DISCUSSION TITLE
[npm-diff] Ignore end of line?

### DISCUSSION THREAD

**Discussion by oBusk** [npm/feedback#635] [CLI] [answered]
I've run in to issues where I'm trying to compare my local build of a package with a published version, where the published version was published from windows while I'm on unix, or vice versa, where the diff becomes useless because every line is different.

I feel like there could be a flag to ignore end of line characters in the diff, similar to the `--diff-ignore-all-space`. I was thinking of how tricky it would be to implement, but from what I can see, [diff](npmjs.com/diff), that `libnpmdiff` uses, doesn't really have an option to do that so it became a bit tangly.. I don't know if any such flag in `diff` would make sense, maybe `libnpmdiff` would unify the lineendings before running `diff` on them. I don't know

Is this such a unusual problem? Would it be denied if a PR was made?

**Comment by MylesBorins** [comment, marked as answer]
hey @oBusk. If you would like to propose a new feature or change to the CLI you should open an issue or RFC over at https://github.com/npm/rfcs

This seems like the kind of feature that is definitely worth discussion!

**Reply by oBusk** [reply]
Ok I gave it a good go at least! 

npm/rfcs#522
